fw/Core/Hitcon/Logic/XBoardRecvFn.h

```diff
@@ -11,10 +11,11 @@ enum RecvFnId {
   SNAKE_RECV_ID,
   TEST_APP_RECV_ID,
   TETRIS_RECV_ID,
-  IR_RECV_ID,
```

Comment:
We might not be able to remove this line, because the enum is numbered and there are existing boards that uses  `IR_RECV_ID`. We might have to mark it as `// DISABLED/DEPRECATED` if we want to remove it.

Reply:
Oh, I see, then this should be fine, thanks for the clarifications.